docs/_api/compositecontroller.md

```diff
@@ -368,3 +369,7 @@ response because they're in different forms.
 Instead, you should think of each entry in the list of `children` as being
 sent to [`kubectl apply`][kubectl apply].
 That is, you should [set only the fields that you care about](/api/apply/).
+
+Note that your webhook handler must return a response with a status code of `200`
+to be consider successful. Metacontroller will wait for a response for up to the
```

Comment:
typo: "considered" (same in the other file)

Reply:
Fixed